src/CalcViewModel/ApplicationViewModel.h

```diff
@@ -64,6 +67,48 @@ namespace CalculatorApp
                 }
             }
 
+            static property Platform::String ^ LaunchedLocalSettings
+            {
+                Platform::String ^ get()
+                {
+                    return Platform::StringReference(L"calculatorAlwaysOnTopLaunched");
+                }
+            }
+
+            static property Platform::String ^ WidthLocalSettings
+            {
+                Platform::String ^ get()
+                {
+                    return Platform::StringReference(L"calculatorAlwaysOnTopLastWidth");
+                }
+            }
+
+            static property Platform::String ^ HeightLocalSettings
+            {
+                Platform::String ^ get()
+                {
+                    return Platform::StringReference(L"calculatorAlwaysOnTopLastHeight");
+                }
+            }
+
+            property bool IsAlwaysOnTop
+            {
+                bool get()
+                {
+                    return m_isAlwaysOnTop;
+                }
+                void set(bool value)
+                {
+                    if (m_isAlwaysOnTop != value)
+                    {
+                        m_isAlwaysOnTop = value;
+                        RaisePropertyChanged(L"IsAlwaysOnTop");
```

Comment:
NIT: `RaisePropertyChanged(nameof(IsAlwaysOnTop));`

This way PropertyChanged will still work even if you refactor/rename this property!